### PR TITLE
Add confirmdialog to storybook

### PR DESCRIPTION
**Pull request by MickhaelSh**
Added confirmdialog to storybook

### CODE REVIEW

File: src/stories/ConfirmDialog.stories.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import ConfirmDialog from '~/components/confirm-dialog/ConfirmDialog'
+import i18n from '../plugins/i18n'
```
Comment (tarasSanotskyi) [inline src/stories/ConfirmDialog.stories.js:2]: use alias ~ instead of dots ..

**Review by tarasSanotskyi — APPROVED** [review]

File: src/stories/ConfirmDialog.stories.js (hunk 1 above)
Comment (MickhaelSh) [reply src/stories/ConfirmDialog.stories.js]: fixed

**Review by maarekua — APPROVED** [review]

File: src/stories/ConfirmDialog.stories.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import ConfirmDialog from '~/components/confirm-dialog/ConfirmDialog'
+import i18n from '~/plugins/i18n'
```
Comment (rostyslaw27) [inline src/stories/ConfirmDialog.stories.js:2]: why do you need i18n here?

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

File: src/stories/ConfirmDialog.stories.js (hunk 2 above)
Comment (MickhaelSh) [reply src/stories/ConfirmDialog.stories.js:2]: without it text on buttons is common.no and common.yes

Comment (rostyslaw27) [reply src/stories/ConfirmDialog.stories.js:2]: ok

**Review by rostyslaw27 — APPROVED** [review]

File: src/stories/ConfirmDialog.stories.js (hunk 2 above)
Comment (rostyslaw27) [reply src/stories/ConfirmDialog.stories.js:2]: import i18n in ./storybook/preview.js

**Review by rostyslaw27 — CHANGES_REQUESTED** [review]

**Review by maarekua — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Review by rostyslaw27 — APPROVED** [review]

**Pull request merged**